petclinic-frontend/src/features/customers/components/UpdatePetForm.tsx

```diff
@@ -89,13 +111,39 @@ const UpdatePetForm: React.FC = (): JSX.Element => {
     }
   };
 
+  // Handle delete pet
+  const handleDelete = async (): Promise<void> => {
+    if (petId) {
+      try {
+        const response = await deletePet(petId);
+        if (response.status === 200) {
+          navigate(`/customers/${response.data.ownerId}`); // Redirect after deletion
+        } else {
+          console.error('Error deleting pet');
```

Comment:
This error handling works perfectly fine, but as a nitpick, it doesn't give enough information. To make error faster to find, I would print out things like the status that it did get instead of 200, maybe print the endpoint it tried to reach, etc. But otherwise it's good